### PR TITLE
Karan add navbar

### PR DESCRIPTION
**Pull request by karanasthana**

### CODE REVIEW

File: src/Commons/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Navbar from 'react-bootstrap/Navbar'
+import Nav from 'react-bootstrap/Nav'
+
+class NavBar extends React.Component {
+render() {
+      return (
+          <div>
+         <Navbar bg="light" expand="lg">
```
Comment (ayu-ayushisood) [inline src/Commons/NavBar.js:8]: Why are you adding one extra node in the tree when simply you can enclose it in <Navbar>? You can remove this div.

File: src/index.js [hunk 2]
```diff
@@ -2,18 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import reportWebVitals from './reportWebVitals';
 
-import { BrowserRouter } from 'react-router-dom'
-
 const routs = (
```
Comment (ayu-ayushisood) [inline src/index.js]: if you want you can correct the spelling of routes variable, else it's fine.

Comment (karanasthana) [reply src/index.js]: ok

File: src/Commons/NavBar.js (hunk 1 above)
Comment (karanasthana) [reply src/Commons/NavBar.js:8]: ok

**Review by ayu-ayushisood — APPROVED** [review]

**Pull request merged**